[quant][graphmode][refactor] Add registerQParams function

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #30553 [quant][graphmode] `insert_quant_dequant` pass support shared class types
* #30860 [quant][graphmode][refactor] Factor out getInvokedMethod in `InsertQuantDeQuantHelper`
* #30859 [quant][graphmode][refactor] getQParams return a dictionary of qparams
* #30858 [quant][graphmode] Remove `values_to_quantize_`
* **#30552 [quant][graphmode][refactor] Add registerQParams function**

Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18818653](https://our.internmc.facebook.com/intern/diff/D18818653)

jerryzh168 committed:
[quant][graphmode][refactor] Add registerQParams function

Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][refactor] Add registerQParams function"


Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][refactor] Add registerQParams function"


Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][refactor] Add registerQParams function"


Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][refactor] Add registerQParams function"


Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][refactor] Add registerQParams function"


Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][refactor] Add registerQParams function"


Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][refactor] Add registerQParams function"


Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][refactor] Add registerQParams function"


Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][refactor] Add registerQParams function"


Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

ZolotukhinM reviewed (approved):
Ah, that's what I asked for in previous review! Thanks, looks good!

jerryzh168 committed:
Update on "[quant][graphmode][refactor] Add registerQParams function"


Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][refactor] Add registerQParams function"


Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][refactor] Add registerQParams function"


Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][refactor] Add registerQParams function"


Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18818653](https://our.internmc.facebook.com/intern/diff/D18818653)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][refactor] Add registerQParams function"


Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18818653](https://our.internmc.facebook.com/intern/diff/D18818653)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][refactor] Add registerQParams function"


Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18818653](https://our.internmc.facebook.com/intern/diff/D18818653)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][refactor] Add registerQParams function"


Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18818653](https://our.internmc.facebook.com/intern/diff/D18818653)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][refactor] Add registerQParams function"


Summary:
For upcoming changes to support quantizing shared class type

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18818653](https://our.internmc.facebook.com/intern/diff/D18818653)

[ghstack-poisoned]
